README.md

```diff
@@ -3,104 +3,58 @@
+#### Note
+This Readme provides basic installation and usage information.
+For the complete documentation, see the [Go SDK Guide](https://cloudinary.com/documentation/go_integration).
 
-## Setup
+## Table of Contents
+- [Key Features](#key-features)
+- [Version Support](#Version-Support)
+- [Installation](#installation)
+- [Usage](#usage)
+    - [Setup](#Setup)
+    - [Transform and Optimize Assets](#Transform-and-Optimize-Assets)
 
-### Installation
 
-To install the Cloudinary Go SDK, use the `go get` command:
+## Key Features
+- [Transform](https://cloudinary.com/documentation/go_media_transformations) assets.
+- [Asset Management](https://cloudinary.com/documentation/go_asset_administration).
+- [Secure URLs](https://cloudinary.com/documentation/video_manipulation_and_delivery#generating_secure_https_urls_using_sdks).
 
-```
-go get github.com/cloudinary/cloudinary-go
-```
-
-### Configuration
-
-For requests to our secure APIs (e.g., image uploads, asset administration) you must have the `APIKey` and `APISecret` parameters set.
-You can find your account-specific configuration credentials in the **Dashboard** page of the [account console](https://cloudinary.com/console).
-
-Setting your `CloudName`, `APIKey` and `APISecret` parameters can be done by initializing the Cloudinary object, or by
-using the CLOUDINARY_URL environment variable / system property.
-
-The entry point of the library is the Cloudinary struct.
-
-```go
-cld, _ := cloudinary.New()
-```
-
-Here's an example of setting the configuration parameters programatically:
-```go
-cld, _ := cloudinary.NewFromParams("n07t21i7", "123456789012345", "abcdeghijklmnopqrstuvwxyz12")
-```
-
-You can also set the configuration parameters by providing the CLOUDINARY_URL value:
-```go
-cld, _ := cloudinary.NewFromURL("cloudinary://123456789012345:abcdeghijklmnopqrstuvwxyz12@n07t21i7")
-```
-
-**Learn more**: [Go configuration](https://cloudinary.com/documentation/go_integration#configuration)
-
-
-### Update 
-
-To update the Cloudinary Go SDK to the latest version, use the `go get` command with the `-u` option:
-
-```
-go get -u github.com/cloudinary/cloudinary-go
-```
-
-### Logging
-
-Cloudinary SDK logs errors using standard `go log` functions.
 
-For details on redefining the logger or adjusting the logging level,  see [Logging](logger/README.md).
 
-## Usage
+## Version Support
 
+| SDK Version | Go 1.13.x | Go 1.14.x | Go 1.15.x | Go 1.16.x | Go 1.17.x |
+|-------------|-----------|-----------|-----------|-----------|-----------|
```

Comment:
Does it make sense to drop the .x at the end?

Does it make sense to have a single column marked as `> Go 1.13`?